Makefile

```diff
@@ -239,6 +239,10 @@ cron/internal/bq/data-transfer.docker: cron/internal/bq/Dockerfile $(CRON_TRANSF
 			--tag $(IMAGE_NAME)-bq-transfer && \
 			touch cron/internal/bq/data-transfer.docker
 
+build-attestor: ## Runs go build on scorecard attestor
```

Comment:
Trying to do a `make build-attestor` is failing for me. It complains about needing a `go mod tidy`. Can you include the resulting `go.mod` and `go.sum` changes here?

Reply:
Is it worth adding `build-attestor` to `build-targets`?

Reply:
Good catch, thanks! Added to build-targets as well